init.php

```diff
@@ -337,30 +371,34 @@ function renderMarkers(data) {
         closeButton: false,
       };
 
-      const markers = data.map(function (merchant) {
-        const marker = L
-          .marker(
-            [merchant.lat, merchant.lng],
-            {
-              merchantId: merchant.id,
-              icon: markerIcon,
-            },
-          )
-          .on('click', onMarkerClick);
-
-        if (!isMobile) {
-          marker.bindPopup('', popupOptions)
-        }
-
-        return marker;
-      });
+      const markers = data
+        .filter(function(merchant) {
+          return merchant.lat && merchant.lng;
+        })
+        .map(function(merchant) {
+          const marker = L
+            .marker(
+              [merchant.lat, merchant.lng],
+              {
+                merchantId: merchant.id,
+                icon: markerIcon,
+              },
+            )
+            .on('click', onMarkerClick);
+
+          if (!isMobile) {
+            marker.bindPopup('', popupOptions)
```

Comment:
just for consistency with other lines
```suggestion
            marker.bindPopup('', popupOptions);
```